### PR TITLE
[DOC] Update example for EmberArray#sortBy

### PR DESCRIPTION
**Pull request by mukilane**
For #18228

Updated the example to include sortBy with multiple keys.

### CODE REVIEW

File: packages/@ember/-internals/runtime/lib/mixins/array.js [hunk 1]
```diff
@@ -1168,9 +1168,18 @@ const ArrayMixin = Mixin.create(Enumerable, {
...
-   colors.sortBy('name'); // [{name: 'blue'}, {name: 'green'}, {name: 'red'}]
-    ```
+   let colors = [
+     {name: 'red',  weight: 500 },
+     {name: 'green', weight: 600 },
+     {name: 'blue', weight: 500 }
+    ];
+   
+   colors.sortBy('name');
+   // [{name: "blue", weight: 500}, {name: "green", weight: 600}, {name: "red", weight: 500}]
```
Comment (btecu) [inline packages/@ember/-internals/runtime/lib/mixins/array.js]: Should use `'` consistently

Comment (mukilane) [reply packages/@ember/-internals/runtime/lib/mixins/array.js]: I've updated it.

File: packages/@ember/-internals/runtime/lib/mixins/array.js [hunk 2]
```diff
@@ -1168,9 +1168,18 @@ const ArrayMixin = Mixin.create(Enumerable, {
     You may provide multiple arguments to sort by multiple properties.
 
     ```javascript
-   let colors = [{name: 'red'}, {name: 'green'}, {name: 'blue'}];
-   colors.sortBy('name'); // [{name: 'blue'}, {name: 'green'}, {name: 'red'}]
-    ```
+   let colors = [
+     {name: 'red',  weight: 500 },
```
Comment (locks) [inline packages/@ember/-internals/runtime/lib/mixins/array.js]:
```suggestion
     { name: 'red', weight: 500 },
```

File: packages/@ember/-internals/runtime/lib/mixins/array.js [hunk 3]
```diff
@@ -1168,9 +1168,18 @@ const ArrayMixin = Mixin.create(Enumerable, {
...
+     {name: 'green', weight: 600 },
```
Comment (locks) [inline packages/@ember/-internals/runtime/lib/mixins/array.js]:
```suggestion
     { name: 'green', weight: 600 },
```

File: packages/@ember/-internals/runtime/lib/mixins/array.js [hunk 4]
```diff
@@ -1168,9 +1168,18 @@ const ArrayMixin = Mixin.create(Enumerable, {
...
+     {name: 'blue', weight: 500 }
```
Comment (locks) [inline packages/@ember/-internals/runtime/lib/mixins/array.js]:
```suggestion
     { name: 'blue', weight: 500 }
```

**Review by locks — CHANGES_REQUESTED** [review]

**Comment by mukilane** [comment]
@locks Have updated the requested changes.

**Review by locks — APPROVED** [review]
Thanks for the help!

**Pull request merged**